lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1628,7 +1630,7 @@ private function validateAndCompleteTypedAssociationMapping(array $mapping): arr
      *     enumType?: class-string,
      * } $mapping The field mapping to validate & complete.
      *
-     * @return mixed[] The updated mapping.
+     * @return FieldMapping The updated mapping.
```

Comment:
FieldMapping type has a compulsory array-key `columnName`. I am not sure that this is satisfied here, as the function does not set it and @param $mapping does not specify it.

Reply:
It's satisfied right below the comment "Complete fieldName and columnName mapping"

Reply:
Thats a different function tho `validateAndCompleteFieldMapping`. The change you made is for `validateAndCompleteTypedAssociationMapping` and that one does NOT return `columnName` in the array

Reply:
sorry, my bad I was working on some badly lined version in my local repo.

Reply:
No worries, and thanks for reviewing 👍